common/models/user.js

```diff
@@ -7,6 +7,41 @@ var config = require('../../server/config.json');
 var path = require('path');
 
 module.exports = function(User) {
+
+//send email again incase of unverified email
+User.triggerEmail = function(userId, cb) {
```

Comment:
There is already a built-in method exposed at `POST /api/users/:id/verify`, it was added by https://github.com/strongloop/loopback/pull/3344. IMO, the example should be using this built-in method, that's why we added it.

cc @ebarault